cas-server-webapp/src/main/webapp/WEB-INF/casComponentsMapping.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:util="http://www.springframework.org/schema/util"
```

Comment:
Cleaner. Deployers don't have to have this alias mappings "polluting" their deployerConfigContext. I personally DO prefer the cleaner/thinner deployer context

On Fri, Apr 8, 2016 at 06:53, Misagh Moayyed <notifications@github.com> wrote:
In cas-server-webapp/src/main/webapp/WEB-INF/casComponentsMapping.xml [https://github.com/Jasig/cas/pull/1676#discussion_r59008790] :

> @@ -0,0 +1,29 @@
> +<?xml version="1.0" encoding="UTF-8"?>
> +<beans xmlns="http://www.springframework.org/schema/beans"
> +       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:util="http://www.springframework.org/schema/util"


Why a separate file?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub [https://github.com/Jasig/cas/pull/1676/files/9c8a4b49aee43d91c0113ec6f61045218cf9e71f#r59008790]